writer.go

```diff
@@ -96,3 +102,45 @@ func MultiLevelWriter(writers ...io.Writer) LevelWriter {
 	}
 	return multiLevelWriter{lwriters}
 }
+
+// TestWriter is a writer that writes to testing.TB.
+type TestWriter struct {
+	TB testing.TB
+
+	// Frame skips caller frames to capture the original file and line numbers.
+	Frame int
+}
+
+// NewTestWriter creates a writer that logs to the testing.TB.
+func NewTestWriter(t testing.TB) TestWriter {
```

Comment:
Instead of importing `testing`, can we define a more restricted interface here?

Reply:
Changed to `TestingLog` interface of the subset of `Log` methods.